### ISSUE TITLE
How to use my point prompt for fine tuning and inference?

### ISSUE THREAD

**Issue by HeeseongEom** [uni-medical/SAM-Med3D#64]
Hello, mr. adrianzzk, I have a question about how to use my point prompts.
I think there is no description or dataset-preparing for prompts.
So, I want to ask can i use my point prompts for fine-tuning and inference? and if then, could you give me a short knowhow to use them?

Thanks a lot.

**Comment by adrianzzk** [comment]
hi ,HeeseongEom!
I'm so sorry that i don't know much about the Prompt module.And i also dont understand whats your mean about ```point prompts```.Do you mean a new method of prompts?
In the paper,the author says that the Prompt method he uses is based on mask random generation. Using the initially obtained segmentation mask to keep iterating as a new Prompt. that's why during the prediction process, you will get four different prediction files (of course, this is my understanding, due to time constraints, I didn't read through this code, so this is just my guess)
At the same time, I personally don't think that this Prompt method is in line with what SAM was originally meant to be, but it does seem like a more reasonable way to automate the training, otherwise you would need to keep clicking on the points yourself to generate the prompts during the training process.
If you mean to utilize a your own prompt method, you can try the results for comparing, I don't have too much to suggest for that.
Sorry again that nothing can help you. Good luck and have a good day.:)

**Comment by blueyo0** [comment]
Seems this issue is solved. For more timely help., please refer to https://github.com/uni-medical/SAM-Med3D?tab=readme-ov-file#-discussion-group
